cloud/ec2_util.go

```diff
@@ -297,6 +299,14 @@ type Terms struct {
 	}
 }
 
+// format /dev/sd[f-p][1-6] taken from https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/device_naming.html
+func generateDeviceNameForVolume() string {
+	letters := "fghijklmnop"
+	rand.Seed(time.Now().Unix())
+
+	return fmt.Sprintf("/def/sd%c%d", letters[rand.Intn(len(letters))], rand.Intn(5)+1)
```

Comment:
s/def/dev/

Reply:
This actually was a typo, but I think it's still /dev/sd...